packages/pwa/app/partials/product-view/index.jsx

```diff
@@ -89,7 +89,7 @@ const ProductView = ({
         onClose: onAddToCartModalClose
     } = useAddToCartModalContext()
     const theme = useTheme()
-
+    const [optionsMessage, toggleOptionsMessage] = useState(false)
```

Comment:
nit, optionsMessage sounds like a string type variable, but it's actually a boolean, what about showOptionMessage?

Reply:
Yes, that makes sense to me. I will update the variable name to match that. Is everything else okay?